hello.template

```diff
@@ -37,10 +38,16 @@ def kill_server():
     bokeh_process2.kill()
 
 
-class IndexForm(FlaskForm):
+@app.route('/autocomplete', methods=['GET'])
+def autocomplete():
+    search = request.args.get('q')
     filelist = sorted(glob.glob(os.path.expanduser('~/data/*.ssw11.hhr')))
-    filename = SelectField('filename',
-                choices = [(os.path.basename(f).split('.')[0], os.path.basename(f).split('.')[0]) for f in filelist])
+    orflist = [os.path.basename(f).split('.')[0].upper() for f in filelist]
```

Comment:
Is it worth defining this up-front rather than on each request? I guess it depends on whether the list might change as the server is running, and how long it takes to read the folder; probably a minor optimisation at best. Similarly for using a generator expression instead of a list comprehension.

Reply:
The file list should change extremely rarely while the server is running, but I cannot guarantee that it never will. It also takes almost no time to create it, so I'd leave it as is for now.